9_ninth_hw/1_first_task.py

```diff
@@ -0,0 +1,108 @@
+"""
+Create a class describing any company. For example, Toshiba or Apple
+
+I will evaluate the completeness of the described class and will withdraw points for everything unnecessary.
+I want to see clean code. I expect to see clean code with annotations.
+So far, no first-class connections with the second. In all methods, I expect to see docstrings with a sane description.
+"""
+from datetime import date, timedelta, datetime
+
+
+class Apple:
+    BRAND = "Apple"
+
+    def __init__(self, __website: str = "apple.com", __address: str = "silicone str"):
+        self.__website = __website
+        self.__address = __address
+        self.employees = 0
```

Comment:
make it protected at least. Try to not use public variables at all . Better private and create properties for this var